projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -508,12 +531,17 @@ export class IgxOverlayService implements OnDestroy {
         }
     }
 
-    private getOverlayById(id: string): OverlayInfo {
+    /** @hidden */
```

Comment:
If this is intended to be internal, mark it `@internal`

Reply:
added :)